dask/dataframe/io/csv.py

```diff
@@ -300,11 +300,15 @@ def read_pandas(reader, urlpath, blocksize=AUTO_BLOCKSIZE, collection=True,
                          "`dd.{0}`. To achieve the same behavior, it's "
                          "recommended to use `dd.{0}(...)."
                          "head(n=nrows)`".format(reader_name))
-    if isinstance(kwargs.get('skiprows'), list):
+    if isinstance(kwargs.get('skiprows'), int):
+        skiprows = lastskiprow = firstrow = kwargs.get('skiprows')
+    elif kwargs.get('skiprows') is None:
+        skiprows = lastskiprow = firstrow = 0
+    elif not callable(kwargs.get('skiprows')):
```

Comment:
Sorry, one more comment/question: In what context is skiprows callable?

Reply:
The pandas library did it that way, but here it is not necessary.